cli/cfncluster/config_sanity.py

```diff
@@ -33,6 +33,35 @@ def check_resource(region, aws_access_key_id, aws_secret_access_key, resource_ty
         except ClientError as e:
             print('Config sanity error: %s' % e.response.get('Error').get('Message'))
             sys.exit(1)
+    if resource_type == 'EC2IAMRoleName':
+        try:
+            iam = boto3.client('iam', region_name=region,
+                                aws_access_key_id=aws_access_key_id,
+                                aws_secret_access_key=aws_secret_access_key)
+
+            arn = iam.get_role(RoleName=resource_value).get('Role').get('Arn')
+            accountid = boto3.client('sts').get_caller_identity().get('Account')
+
+            iam_policy = [(['ec2:DescribeVolumes', 'ec2:AttachVolume', 'ec2:DescribeInstanceAttribute', 'ec2:DescribeInstanceStatus', 'ec2:DescribeInstances'], "*"),
+                        (['dynamodb:ListTables'], "*"),
+                        (['sqs:SendMessage', 'sqs:ReceiveMessage', 'sqs:ChangeMessageVisibility', 'sqs:DeleteMessage', 'sqs:GetQueueUrl'], "arn:aws:sqs:%s:%s:cfncluster-*" % (region, accountid)),
+                        (['autoscaling:DescribeAutoScalingGroups', 'autoscaling:TerminateInstanceInAutoScalingGroup', 'autoscaling:SetDesiredCapacity'], "*"),
+                        (['cloudwatch:PutMetricData'], "*"),
+                        (['dynamodb:PutItem', 'dynamodb:Query', 'dynamodb:GetItem', 'dynamodb:DeleteItem', 'dynamodb:DescribeTable'], "arn:aws:dynamodb:%s:%s:table/cfncluster-*" % (region, accountid)),
+                        (['sqs:ListQueues'], "*"),
+                        (['logs:*'], "arn:aws:logs:*:*:*")]
+
+            for actions, resource_arn in iam_policy:
+                response = iam.simulate_principal_policy(PolicySourceArn=arn, ActionNames=actions, ResourceArns=[resource_arn])
```

Comment:
same here, iam::simulate_principal_policy api call is newly added. Please make sure CfnclusterUserPolicy is updated

Reply:
will update docs